src/Components/BookList.js

```diff
@@ -3,14 +3,15 @@ import PropTypes from 'prop-types';
 import Book from './Book';
 
 const BookList = (props) => {
-  const { books } = props;
+  const { books, removeHandler } = props;
   return (
     <ul className="booklist">
       {
           books.map((bookElment) => (
             <Book
               key={bookElment.id}
               bookdata={bookElment}
+              removeBookFromList={removeHandler}
             />
```

Comment:
- [ ] List of books need to  display books from Redux store by using useSelector function.

Reply:
- [ ] On category page please use redux that when the button is clicked it displays **under construction**